WalletWasabi.Fluent.Generators/OaphGenerator.cs

```diff
@@ -0,0 +1,178 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace WalletWasabi.Fluent.Generators;
+
+[Generator]
+public class OapthGenerator : ISourceGenerator
```

Comment:
Is the name of the class correct? or is it supposed to be `OaphGenerator` without `t`?